src/components/organisms/ActionsPane/ActionsPane.tsx

```diff
@@ -91,8 +91,33 @@ const ActionsPane = (props: {contentHeight: string}) => {
   const isActionsPaneFooterExpanded = useAppSelector(state => state.ui.isActionsPaneFooterExpanded);
   const kubeconfigPath = useAppSelector(state => state.config.kubeconfigPath);
   const [key, setKey] = useState('source');
+  const [isButtonShrinked, setButtonCollapsedState] = useState<boolean>();
+
   const dispatch = useAppDispatch();
 
+  // Could not get the ref of Tabs Component
+  const tabsList = document.getElementsByClassName('ant-tabs-nav-list')[0];
+  const extraButton = useRef<any>();
+
+  const getDistanceBetweenComponents = () => {
+    const tabsListEl = document.getElementsByClassName('ant-tabs-nav-list')[0].getBoundingClientRect();
```

Comment:
Could the method return an empty array or undefined? Maybe add a check for this before getting the client rectangle

Reply:
Sure!

Reply:
Done. I check if tabsList is a non-empty array in useEffect